src/styles/_header.scss

```diff
@@ -25,7 +25,7 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
-    transition: 400ms cubic-bezier(0,.13,.52,-0.79);
+    transition: 400ms;
```

Comment:
im not sure that it's good to write only delay here without other properties, i mean name of transforming, duration, type (ease, linear etc.) . Moreover i think it will be proper to write every property of transform instead of "all" (just for future fixes)